doc/docs.md

```diff
@@ -604,33 +591,73 @@ numbers := {
 }
 ```
 
-## Module imports
+## Modules
+
+Simple programs don't need to have a module name - it defaults to 'main'.
+Every file in the root of a folder is part of the same module.
+For information about creating a named module, see [Modules 2](#modules-2).
+
+### Symbol visibility
+
+```v
+pub fn public_function() {
+}
+
+fn private_function() {
+}
+```
+
+Functions are private (not exported) by default.
+To allow other modules to use them, prepend `pub`. The same applies
+to constants and types. 
 
-For information about creating a module, see [Modules](#modules)
+Note: `pub` can only be used from a named module.
 
-### Importing a module
+### Module imports
 
-Modules can be imported using the `import` keyword.
+Modules can be imported using the `import` keyword:
 
 ```v
 import os
 
 fn main() {
-    name := os.input('Enter your name:')
+    // read text from stdin
+    name := os.input('Enter your name: ')
     println('Hello, $name!')
 }
 ```
+This program can use any public definitions from the `os` module, such 
+as the `input` function. See the [standard library](https://modules.vlang.io/)
+documentation for a list of common modules and their public symbols.
+
+By default, you have to specify the module prefix every time you call an external function.
+This may seem verbose at first, but it makes code much more readable
+and easier to understand - it's always clear which function from
+which module is being called. This is especially useful in large code bases.
+
+#### Specific imports
 
-When using constants from other modules, the module name must be prefixed. However,
-you can import functions and types from other modules directly:
+You can also import specific functions and types from modules directly:
 
 ```v
 import os { input }
 import crypto.sha256 { sum }
 import time { Time }
 ```
+Note: This is not allowed for constants - they must always be prefixed.
 
-### Module import aliasing
+You can import several specific symbols at once:
+
+```v
+import os { input, user_os }
+
+name := input('Enter your name: ')
+println('Name: $name')
+os := user_os()
+println('Your OS is ${os}.')
+```
+
+#### Module import aliasing
```

Comment:
Might be worth moving this subsection in another PR (e.g. under Modules 2) because it is more advanced.

Reply:
I think that should be in the regular `Importing a module` section.  It's better to keep the information for a subject close together, to avoid having to search all over the place to find pieces.  If you want to keep them separate, then the first section should have a link to the 2nd, or people might think the first section is all there is on that subject.